### PR TITLE
CX-10233 : Screen Readers: Resize Modal button is announced without sufficient context

### PR DESCRIPTION
**Pull request by iruchipriya**
## Description

Story - CX-10233
Prerequisites:

1. Tab to reach the Webex button and press enter to activate it.
2. Tab to reach the resize modal button and observe the announcement when it is full-screen mode and half-screen mode.
**Expected Result:**
The resize Modal button should be announced with sufficient context i.e. When the model is in halfscreen mode, it should announce as 'Halfscreen Resize Modal' and when the model is in fullscreen mode, it should announce as 'Fullscreen Resize Modal' or similar.
**Actual Result:**
The 'Resize Modal' button is announced without sufficient context. i.e.
- the 'Resize Modal' button announces the same label for both the half-screen and full-screen mode and the screen reader user will not know the state of the control as 'Resize Modal, button'

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other... Please describe:
fix(accessibility): implement dynamic label for 'Resize Modal'

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Issue screenshot - 
image
 
 Video Link For Issue - https://platform.applause.com/services/links/v1/external/e418790303d083cd604a5701204ac017906e12812bec60a519e8b183979279e9
 
 Screenshot after Fix - 
 
image

image

VidCast Link - https://app.vidcast.io/share/d5f69e35-c744-4a5b-9c72-f25d5f7fb263

### CODE REVIEW

File: web-components/src/components/floating-modal/FloatingModal.ts [hunk 1]
```diff
@@ -312,7 +312,7 @@ export namespace FloatingModal {
                   ${!this.minimize ? html` <md-button
                     color="color-none"
                     class="md-floating__resize"
-                    aria-label="${this.resizeAriaLabel}"
+                    aria-label="${this.full ? 'Fullscreen Resize Modal' : 'Halfscreen Resize Modal'}"
```
Comment (merajcal) [inline web-components/src/components/floating-modal/FloatingModal.ts:315]: is this maximize and restore?
change the text

'Maximize the modal' : 'Restore the maximised modal', But this should not hard coded inside the component. the localized message should be passed by client code e.g. desktop

**Review by merajcal — CHANGES_REQUESTED** [review]

**Comment by vinitkumargoel** [comment]
@iruchipriya  You need to update momentum version also

File: web-components/src/components/floating-modal/FloatingModal.ts [hunk 2]
```diff
@@ -193,8 +193,18 @@ export namespace FloatingModal {
       }
     }
 
-    handleToggleExpandCollapse() {
+    handleToggleExpandCollapse(event: Event) {
       this.full = !this.full;
+      this.dispatchEvent(
+        new CustomEvent("floating-modal-fullscreen", {
```
Comment (merajcal) [inline web-components/src/components/floating-modal/FloatingModal.ts:199]: rename event to maximize-restore-event

**Review by merajcal — CHANGES_REQUESTED** [review]

File: web-components/src/components/floating-modal/FloatingModal.ts (hunk 2 above)
Comment (iruchipriya) [reply web-components/src/components/floating-modal/FloatingModal.ts]: updated

**Review by merajcal — APPROVED** [review]

File: web-components/src/components/floating-modal/FloatingModal.ts [hunk 3]
```diff
@@ -26,6 +26,7 @@ export namespace FloatingModal {
     @property({ type: Boolean, reflect: true, attribute: "full-screen" }) full = false;
     @property({ type: String, attribute: "close-aria-label" }) closeAriaLabel = "Close Modal";
     @property({ type: String, attribute: "resize-aria-label" }) resizeAriaLabel = "Resize Modal";
+    @property({ type: String, attribute: "maximize-aria-label" }) maximizeScreenLabel  = "Maximize modal";
```
Comment (rathnavel) [inline web-components/src/components/floating-modal/FloatingModal.ts:29]: Change "modal" to "Modal"

**Review by rathnavel — APPROVED** [review]

**Review by rathnavel — APPROVED** [review]

**Review by merajcal — APPROVED** [review]

**Pull request merged**